### ISSUE TITLE
Items not importing for my PoE character

### ISSUE THREAD

**Issue by RandomString123** [PathOfBuildingCommunity/PathOfBuilding#7997]
### Check version

- [X] I'm running the latest version of Path of Building and I've verified this by checking the [changelog](https://github.com/PathOfBuildingCommunity/PathOfBuilding/blob/master/CHANGELOG.md)

### Check for duplicates

- [X] I've checked for duplicate open **and closed** issues by using the search function of the [issue tracker](https://github.com/PathOfBuildingCommunity/PathOfBuilding/issues?q=is%3Aissue)

### Check for support

- [X] I've checked that the behaviour is supposed to be supported. If it isn't please open a feature request instead (Red text is a feature request).

### What platform are you running Path of Building on?

Windows

### What is the behaviour in-game?

My gear is all equipped and has been for over 15 minutes now. 

### What is the behaviour in Path of Building?

My hand and body armour is not importing correctly from my profile. The hand armor is precursor gauntlets and my body armour is Royal Plate. 

### How to reproduce the issue

Import account name 'randomstring123'
Import settlers character 'MadManSmash'

### Character build code

```shell
eNrtPdty2ziWz62vYLmq9yWyjSsJZpPZ8i2OM_Elkp1M94sLJEGJMUU6JGXHPTX_vgcgJZGyKEu23LXTOzM1Hgk8F5yDcwVA5d3__BzF1p3K8ihN3m_hHbRlqcRPgygZvN-6uvywLbb-52-ddxeyGJ6H--Mo1k_-1vnlnflsxepOxe-3hNiyCpkNVPF1QopeA6lbmRRDlSan8nuaHafB-62zNFFblieTICom3_xY5vmZHKn3W6cyk-NAZVuWzH2VBAezR5_Gg4HKEjkuAGMIcH6hss-a_964SE_TAGBCGedAcCSjpJ_6N6o4ztLxLci1Zd1F6r4EOjm9OO9d1iYXJfXJgXC_vLuI5YPK-oUsrBz-vN_aAx3JgfoYAfc7GY-BDtraXQq7P87y4lCO4OPTOP1bpYIZ2A5hggrOXEaEg2kb1kWmjsJQ-UV0pw6yqDgYysRfgdu6sKfjuIhu40gvzFPwHx-RxqgV-DItZHx40X-abAmZFqtDrkT2YBzHYNErwfZUrrI7WYB9r0Y7HXlRooK9u8EqwGkcpPfJFJLtIGK3mlim5HlYrn1PBtE4P1VFpvKZznd4G-6pTORBmq-gSQ15oTJwxGIthL7yU_DddXmsifk5ClUD0l0KupYgFcKC6RCyQzmzXcYEF4yswHFNsY76q8KtTfh5E-pBAFsNsp-O4xUhi1kkoZy0e9yPOiTmbhvkofo5A4N01E6wAYlaCZ4kMzEwcZYQrEO6vH2Cd6mOHCvGr6OPF1NIB1GH7DDiYsdBuDUXDB_yyJfxqfwZjcYjCMOX8kbN-AneLuznaDAsEoiDbciui1ol-xBlqg3PIcxpj3hx0Irn0lajgOyS5m2InKGlTj2zOoLJkmTsv9XQJ4lfU8EyuldJZrJDLYkv46BReuCDulLwYrUyzoxN5curJNmS2UAlFceHWTBjdMdehvZZKX94DOVUTxZqtWBeW0NnqYI1cF3BS6ku0O82d1bDmFfVtrODORYYbIww7pJlVBbrDdMddxnWmmo7SlQ2eOgPIxUH60FPZnYgb1eqMf23deyVtN9kt0gbK6GuqZJvMgtWS02rQa078zuZ14O13R7-Sq2W8HWFEtZqISpWCjACtWoNfpGl33WVH6-HtpeN0nFWK11cbi8Vo0SoS4GZeCrhlC1OTwVjv5Hg2suxac-yH0OXNieN4zyJBnON40W4rF0VRSH9m8M0GKysPcNlLYzmBPvj21sIQdouViWgUyl0GVG93mRPA5-DUa8UAXTKnaO_JM9OgVemP60j5pi0G9EcxuqS6CJgjss2WQF6ZQ7TxTyFmDGC_GFa9NM0qPVYrDULfIC-cqUm0QCu2KxepPcw-aHeaMnXg4ZyabbirVPJVPLHw8r0G-ArMThKgnGmPWFlHvMYdTaTiHwZjSCg5vmhLKQVVGX2V5lFMimI2fvJlcz84WdY_Q8yjj0IBO-36qP62xwi1oia_rtds9OlP52MbtOssGKZF3u-n46T4qPMh1BZ-wFRPkRKQaGqoIxjyRyfBJzaoRdwx3Ooj5VkWwa1p2Q8gtU5KL8eTLaySlqIhtDlYEkc5iI3II4NFBEmFCEPh76PHMlDCkHBUj_1bC5kVjxMNr7MfM0IiJMXUWL2KiD4xfGW1R-m93vBnVbmZZrG-XS3TN7eqiRo0LjMlLLkJJT5WhdG1fqLNYJZQxIt_QPI_JNSmztdFznsX91_Ypgw6lJOEYFvUGAJ1GUcIfjicCK62LWJgC9MuIJ0uY0p1VgE226XughrQM4p05Alms2py7s2Ew7XzwS3aVdjm2eEC9GlgnHNm7uU8i4R0J7_q7GDeBIYO0hSUAuEVIcR0iWUObhLGBK0i6kLI4zpv0CQ464Nj134K2BWQK7LGTSBXUwItbsutTGMQqfRZS7mtEttB7EukHM4kCIcKBHX6drQtjkwLDg8ZNgFqtQmqFtO2qYMAwzHSEMiEJgi29UzcjHrEu4I3MXMpagLfR98NBrCDGTt2pjYpAu2hrQmhCaAzDzA9OCvI4QDc0LMAWJUiweah4kI-NItdYSRERJjF0YECN7lmIMGEAaBQVJQOGaI27BAHEY4tx34DCVzlyAOmNBLgsTU1X9d7MJHm8Ef4lAAJpR2zVpzRBkC1SEBiuLAHAswC-AiGMgJMsCkXFBstzIGs7SuC8y5K4AI8AYVC6LFAW0Dc2NOWnMgFBZcmwxYGnwGPXYZQQJWhwELsEIbhmENQa0gLJCh1AG1IhdQGbgYWBRCFHRgrAseYoA0hkypZggUQSLiUmDCtPDGJoENTBcWUCvIobCuep4gMSCAVYHFIEa7pQ9At27zrgBbwVzYgOtoqRGYeBf0BFYCFuHaoDSMwWippsFBMwRQwEiIXigQGJp_kABsSwcsvWkjs4e9pmUnEbh3AS5b23wnvNpXL01fe-8v7656n82HX4ZFcZu_3d29v7_fuZXFMA3VTygud_x0tHsLSOD32_lNFMfbmuzuHvxnf3C0H4xy9c121dEfv0cZzWJ2fX52_Ono7HtwdvdZ3N_c_Pb74H7v-OK3vasi-5Iwx8nG2adrNrzoCddJtq-ufz8Jts-GV6M-Po9PvqOH7U_nh4d3lzeS3v0If2z_5vVurrdP_8DiuvdRuWfj8OPfRxd3h-Ne78PBgX99-dE5G38PDoNYfMiKLz3h_Tba3vtuD5gIt0-v_B5Of_79y9eT0-ODHgTf6Ovw-_2X_vXX8eXhp_u723FvhO4-bR-Rr-EX17kaX_HbH59_O7_5PerbX4J7NE7i9Mc33svlJ8dOrq99dk4GdHSwjw4G_g-xT88G6Dr-jgb33vHR-OgquPry9QMv9n8MzoKL8F5-vWI_ZIb_fv0R__Et_8fxKDrOf7_Cv4Gu7vb2xt_O7npidHxyF30TYBLBabaPU4K-XX_N3Ct-etqLP4Rfrr_dXfsPZw8DGp9-Bn6R-h3ov39vFnB3soLvyuONvFzO6puJb3q9TbDasqJCjcrl3y2RdutY73QllEUQEcs0uquDu0l4OvrrD2dpoQysHpx8edfXdpFbOSSfYzXK9x-gdPmgO7zZhrcpZqr0oaH7qijzcB3n_VaRjZVOvKEcx3r8y1jGkc5mqD76uTxiStJsNN3cAlKQzXSBXVK8fLjVgX3v8-fyyV5cVMQ0u0lqK1NYNSErqjmGGTTHRnuzWR_I2M_NvKPEj8eg2qSq6iqXi6WnZ6ZPzPQmTjCRKI9TfQ6i4pEqtuapTpn-8g7mVqEex6knYzKhW0qM3S1rUK7gqSpkAOXN7gksab6rhdo11ODTaRoXKukPlU7yd2X1YnDq48aZH43-qCm8-qxhDkvdbzUmhyfSzQlr6qBq4okcVfVCycYq-exuRt7pah9AAXKelG3ndPdvKvejpxViTQnVyAI6r6cRzcy6H4JSSoZWyXF3o7Zwpu73MrA1f9b5T_Xy6NlUHc3h11NByceqGG3aKvR2ZVE2dlOZq7FWC5jivJ7QExbTEKyn8HTUMVSeFXe-KXmbJhax-vdaskfhR5PYqNF9jQqjsIbmZ4NTnc-GXk_bUx6rWtdKEn7UHU9DvGpkKlv1_fUEKxn82UaEX9-Ipv5r7hjkRRbdNFVdG2_14wbuK2a1Gpu_4FJU1g4zji9TiNL5XF6dDU_1Xx98xcQxLlL1U2Vme_lPU_x-mhb5n6Hwz7DC_ViOmqlrOjhV9mzoFdMV8LAMk81WJpdqBCV6MTvjmgo692Qq7dz464lcMbIqTn96ov4zTOybzOI0C_JTmd00tN98MFV-c_j1dF_x-S_QRP7fuWXYbbow1ETPk-rm3iytTEbbk8oM7xVTCjCxzhNLs9msxx2qQmWjag-4Ifrck6ncc-OvJ3ST0SOHqz5Ck262I8oNB_3RuJyBOElux4Wh-H5rFOX-tTcOQ329E8SB_Kwvrx59-HB0cHny9ag6JqijGJGvk_HI0zcay_-f7pO86ytzkGnlYy8vP-rCUt2bicDcZRTnWrQ4lre5mu6cmx2FauYx4C2hZqBgySe3QxfTmgG0UzoyaTEZgCP5WaRa5zV9_sSkSob6lFwfq7RR0_cw2wmVZ6264S3P-ls0ZW68tlPR905bxdEPl-CejG5l3Mq5evqEJgq9pwTmGoWRr0-kli-53oEqoZboxffHmfQflqx3dVDcTmM_NveEFxMoH7YjX6RRrj17MXb1dIlWB0nUvp7V03b0Q-XLVtnLh-3I0-sjaRmRF1OZQi2hdJaWWz7gNHtRrA92W1f2KFZTkHaC58VQZdVpWBulU4hRE5CljpNF3rhod-MaxBJdmXtuLRrSz9pRywtcLTLoZ0siUeMSU4tC6zDtpMrLP62BbBlqecjfqr_qxsCSJah2wVrUX9u6WqyEyb2hFvmrx0ucxMTfvbs0CsorIy3uMge2LGCk_s3LyZh7MC8nM38x5uUUP8Qyv2ld7-ppO_pVEenCZgGVsoBZiYh2qpdR0L71MgqX0IMU40w9m0BvvhKZ4faW1yDT2xwLkSdPl0WO6pLHsymUV1GejW5uyjwb28R_qIoVSLA0AUxhlvhHMU4OQRnFEt9YkZSZ1uJAMpNuLVplKlwo6doUG6cuy2JA7ZiknRDk8o9LqsXVKE0vfH1UMi6GF2kav4zgozvqLyGm766Ob2USTMidL6rRZ-uwovbSIgea5orTob4i-1IdJmr0sIBQ-7ze7U66OnPrSPdZ1c2sfpHpvZM_0nT0m2lC9afqNJhWJ8BQcR9GsG6ZsbcJHw34j8mtsnemN66Oo_Xn6Wl0O4Fxrso3ZMo9GoMx2ZMx9Oqnxz3orIuHt1Zvr3fUOUqKYSbNO2XW10inuE7fSKMHjh6U9Undq7hzlUQ_xso6OXxrMZe7vkKedALfw74MmOI05Mp3iY05cuABQ9RHSFCkMPOITaVNAF5yz3HdsGPmY_Ty1nJox3zqqR9AGXeg1YkjH1zjrYU6e0GQW9gqUotak8u7k9NIGCzbtrzzRjjma9WtWNDEweQ7bxxmhk0J1DH3qRUEKmVh29IJ0LpVWXVFxfTz0I33ZAKkM_1Xv9PItyq1VR1_KwR5EoI-CTG5Nmu2RprrRprrdnV28uXqqHORRflI6gPKIz-OwGw7l_dQGwyGhbUfy0DV18xmiHFPYc9nAtaESldhipkdOMSjLufc59yxceC6AfOR8AJfBQQzzkNGmI34_JpVlxX0IlVXNN5ax9vw39pqclpfTdxh6Fe975gpCR5mldswjxdN_GqVPaVeO1MLVus8Wfj7YRQr6xAmYH3TZbnGMoZiGytwxLypdAhfwHjenu6jYmiV3pMby-gBbClbB5NFMy9nZXYFDMIxRINkgoJ22K9WGs7YNIUwfR_gSb2FmEiDvx_DYlXob9AOsUbmDUktlLmAb_XNQY5Vmo_G-DaEBnaC8n_HhJdA8Cch7CchnHZHoQsC3L7SFxkO0ixNZKeXPsBa7I-zQZqAzsrYAKbKUPV5H5a4ev0GzAzse8fGrOFJKOS27wWwgJKEEv7HAzdwbUIUEQ488ULXJUwhlwd2SANKJFGYMB54wnew1_QkZ6En9bZ74Eu9mi_ZvBkZ3ziutotRmbBNOOvghpmbCDcNejrt6djXeUPAwQBT38W3yuZSOxuMEzOu79DXx__pZzIsVPCvN5iIWjj9d7K2RZbCmpZyund8cgC5MND3sq3e2HuwTP-hXRhKSesyzYo0yhsxFTHPI9hFns2p5DTwKfdDHHgq9LDtE0xt6XkQPj0WKiodh6lABjR0fJcGyGN-0xJw3RJmC09Zc-FpI4iWtb41eU2ow9z603KpLOhVTC431eIrZim-SKUX-q54pO1PZZNDk2MgqgNXQ8MXsRahrt8w5GHgYOpDnRF6dhC6UikHgYr1a3DU9pVPvdCXWHmeT4lLXZ8HfiCcUBJXOqihX17LWZjXFSyaCib4VyvTrwZN1VumA6iwOtSdPSvVqWffH-okBQ72kI7XUfbKPrBI2fYiZR-msU5a4_h2COtu1FtXqB-6OICETjlBtgh4KELEBZKEOYQKP4B6AIUSyUBbNUM2YoSEiEiGXRU4od1QKESkhQZLnGbWPwBjLECHB5BXla9jEGTQLB0ngVbZldaqs8ykdfKe6FxtUqvPM3JnQYY5TTMvCqyPMksgenR641wXZX2ZD-val1B3ESlFEMISCLBcF0pmOyAUU0dAsvACiqjjMuUHCAUyJEGIQkWkkJIqJOfKZrdeaM2pnJCn653OG0Z0OIeaQiWDYth5Q021fKh-FkrLZkGXZZ0khYLWYKBMinD5o5xTq6o52-GP6uo3Nqvj6FqnnlIEmaUUw7Dab5xUg_-pad5viUWe3veHaSynEfUwuosSVWr_kdsT6ksE9T4Xnq0IZB-hWEB8X7gMhUxQGwoayRzHFZ7r4FB4Ujdsygl9SGwQgZtujxa7PYw3wih228PoKqu6SBPuououkyMP-gF9LN2BIi9LA-B4qffHqvsR0yKP8dYajzKH11WGoZUNgsCXTAjojZSA_xEBaUg6vvRdRUPkOgFRAXRTQSgCYrscNKnfn3ClJ5upHfOWIq_RLDVd-A12HvkaE3Nxst4kTTCm_ozp41IAXHiZB4vH7ovthVWhXaettwCN85bd2jRy_39z3iUQot2oMVpg1ZMbPhAvBzFURfsy8_Sax2Ov0YsEYHoeeDVnygskchhHoU2pD7297TMvVAHnhOiyifuIcJ3Msadf2oL0H_iiYabUbetFrF5rvYQ7GNVqIr2ZVtrD5RBaV4hRAQA07HY-DzUr1lpPbbYCFnQquEHuKDavus718RM65iLGv32vghft2-mfrrH0G1W3nT19G8TaG41jaGnr5uE4FDPMuMcdjH2Pco6hL1U2CgKoPRwPB4FNsa9c4buhYyMkoODwOMaIIk6amz62XY9WZC5clb3hrJhgaK64MIGnUU6Um3umPABss8oLtvb0phF8n751vbg3peJRXPtrRZ-FZkEWVQanMk_9IahoUhv0oxjC8eOyQEisu1XIdJLZNlMMTATZvoB633Y8wQLqSUKEg4TCiCjhMFtwB1otR1AiHTlvHQvLAj5XFuirDhaZVgTm9RLdNklY_Y9RYXm67izjyErrs1Avi3aB-jfjOLbOZBSDu6gCglBh9XSJWdOIC60RCIaZhEYjcBXC0g4cmwpI-bbArk2JzSHC6l1tRwS2bQfKxSr0PexRW7jNCt1urY60v9Ayr-rX_BvBja-yR2qLR0U1lO-L93V0z9rmQBg1d4z01nH542w132LuX9y3nleWY7bAyi60effj6BZqqLzTByu_yW9gyNzM7lTNDRiAgydf5mtR3NxrFL6nMNSc0oV6VGBI7A53CPIQZpTSwPHDkHqucgnCvsA8gO4RCRISIRDGdImL1vL7_vZ-_QxmfsOJzPL79DIPGNGPMViauWk0DdLzVeIbwtptr7mRNfnJiir11yI7_be2voWWwxce5xzK-8QDddxAvQeaTPMcXH9hI0MYbt-udqldN6EAMj6jMgiobyPH9Qj3OWR9JaWHBLMhznEswoASMDQaMKJPgnyhI5pHqCNYw4RErZMhjZOfXnO32pnvZtijVSTNbZ9Jg6ID43xrs42fOhMK08xSuumAaaFmLaHPba0wS0c6v-hYq3UVP5gyMg1Lm23AyrxOwERkgJvZ8FJg2jz0WQbb99PMH5pcF0EqhChxAIs-Od6ZZsZSBf_ppVbqpewFMfkrlKDKGM9xVlbLLelfhNBjYb0l5wiXSKr8wMMC4p7HmQ9ZXjEoo11EXAmdlM09QYX-JYnQC6XrQASeP9lZkv6xWJz-3zAHLTzKNgl_bnOsKrKbZYDeLl0UirEzKw90rz7POv9rF8z6_fm2OxLNF-vjtDC_Q3DhXfU-6-sc5QX8Y308kc9-ogBNJtKGMHkLboZCnkIxVYK15z2Y0F_9RAJfg-e-iot5fLIGfnkgQ2cYfFUpzbt-86zZM9T1EvGnLxK_nMZigewNCGRvQKBn0VgkEF5EyDw6CYDK_O9zlBTNhSCr9pMdE47PMW-6AZWSTaljHYMtfy1jJYWu4a32i2dA1qYwQ6DPY0k3pX--AWPAG_AvvIF5sDUtYQbuvChGrbP-Cz2SbUCB6yed2tJj8iIF0JcqgLzQh9ebQPAwOf98gRGW1clLKCz2bP5iCvamci_eVJBZ3zZrrMXztMBevJZ8M0a1jvDlgULNL_FzYyHdQEihL_RK9kJ8vJkFeHaunOG5q1ltTWB7NYyacbBV51YzD_4ctfKXRusNrQvbVHjZWG1Onxs5a0vyJKLeA2ksIn1eeCKbWYWNZYtNtI3seTKtgWW0XzNgbD9P-xuqOPiLs9TL85y9bjyoaY89K4A85ljuHU1-zcG8jWN-zCFNwmhQvZdTfqnezDFI05HGftJFLH2l70GorJqE0oeb1b-eNfkhBwdNhW1BqP8DWBMs3o5j7qT2zRn6o5-LaOUxsd8pPFnCwOxer8tBSzH5CYkJDl0i-mj6z3h9i_QbfSowHA_0j3T0VRzWOPMnWDd__L-m-ifwqn8sII1js99YX7MnMKevVk8w7CW88mgQxeehebETZDRvp87_zMfu1MTK19DMt7913u3O_2N3_wu9von4
```

### Screenshots

![image](https://github.com/user-attachments/assets/7a0a6f4d-1478-41ea-8418-e1c26eeec12f)

**Comment by Jackaces** [comment]
I think this happens to me because of the new item bases. Those are the only ones it's not importing.

**Comment by LocalIdentity** [comment]
Fixed in the latest update (2.47.0)
